--- httpd/httpd/trunk/modules/ssl/ssl_engine_pphrase.c	2018/07/24 11:27:20	1836546
+++ httpd/httpd/trunk/modules/ssl/ssl_engine_pphrase.c	2018/07/24 11:32:17	1836547
@@ -803,6 +803,7 @@
     
     return ui_method;
 }
+#endif
 
 
 apr_status_t modssl_load_engine_keypair(server_rec *s, apr_pool_t *p,
@@ -810,6 +811,7 @@
                                         const char *certid, const char *keyid,
                                         X509 **pubkey, EVP_PKEY **privkey)
 {
+#if defined(HAVE_OPENSSL_ENGINE_H) && defined(HAVE_ENGINE_INIT)
     const char *c, *scheme;
     ENGINE *e;
     UI_METHOD *ui_method = get_passphrase_ui(p);
@@ -883,5 +885,7 @@
     ENGINE_free(e);
 
     return APR_SUCCESS;
-}
+#else
+    return APR_ENOTIMPL;
 #endif
+}
