From 1125fe84147e947eadb8d2c417cdb2f2125f90f6 Mon Sep 17 00:00:00 2001
From: Michal Karm Babacek <karm@fedoraproject.org>
Date: Thu, 26 Oct 2017 04:49:46 -0700
Subject: [PATCH] CMake MSVC x86_64 fix

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0fe939d..52062f1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -106,6 +106,7 @@ set(ZLIB_PRIVATE_HDRS
     zutil.h
 )
 set(ZLIB_SRCS
+    contrib/masmx64/inffas8664.c
     adler32.c
     compress.c
     crc32.c
-- 
2.9.0.windows.1

